setUpApplicationDefaults.js

```diff
@@ -14,11 +15,19 @@ import httpGracefulShutdown from 'http-graceful-shutdown';
 
 //TODO: better way to handle asynchrony here?
 
-export default async (app, config) => {
+export default async (app, config, existingLogger) => {
     let server;
     let metricsServer;
 
-    app.use(pino({
+    if (existingLogger) {
+      // we should probably check if this is a pino logger, not sure we support other types of loggers
```

Comment:
Correct, for now. Find some field to test it on and I'm good with it .. maybe see how pino tests the logger field in its config

Reply:
theres no obvious field to test. i looked.

theres a `.version` field, but theres no guarantee it wont be defined on other loggers

Reply:
I see that pino-http also just takes anything and doesn't care... so I think we can remove this if-statement -- then good to merge imo